server/web/helpers/authorization.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const User = require('../../models/user');
+
+module.exports = (request, roles_array) => {
+
+  let isAuthorized = false;
+  if (User.highestRole(request.auth.credentials.user.roles) >= User.lowestRole(roles_array) ){
+    isAuthorized = true;
+  }
+  return isAuthorized;
+};
```

Comment:
we don't need it here and even if we needed it you shouldn't put server side helper in handlebars helpers directory.